src/objective-c/examples/BazelBuildSamples/messages.proto

```diff
@@ -0,0 +1,118 @@
+// Copyright 2015 gRPC authors.
```

Comment:
We probably have already discussed this offline, but what's the reason for having a new copy of this file?

Reply:
I will remove those and use the ones in ../RemoteTestClient after we merge this https://github.com/grpc/grpc/pull/19733